addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -818,6 +818,9 @@ registry.backgroundVideo = publicWidget.Widget.extend(MobileYoutubeAutoplayMixin
 
         this.$iframe.removeClass('show');
 
+        // make dynamic snippet available to adjust the iframe
+        this.$target[0].classList.remove('d-none')
```

Comment:
Your commit message explains the problem quite nicely but this solution is not the right one: you should never have a widget definitely impact the rendering of another one. In this case, the backgroundVideo one should not touch the d-none class which is the responsability of the dynamic snippet one. Also, I am not sure what guarantees you have at this specific moment that the iframe can be adjusted properly since, as you explained, the dynamic snippet is being restarted.

As you explained, the iframe is readjusted on window resize. An ok solution could be to trigger a window resize once the the dynamic snippet is ready, making sure the iframe gets adjusted at the right time. To be more cautious in stable, we could rather trigger a custom event on the snippet, and the background video snippet listen to that one.

In master, another more generic solution could be to trigger a resize on the window each time all widgets have been restarted. That way we would ensure that any significant change in the page trigger resize handlers (which make sense) and only once (not potentially one per many snippets).

Also: please configure your linter properly, your only line of diff is missing a `;` at the end :grimacing: